awx/main/scheduler/task_manager.py

```diff
@@ -253,13 +253,11 @@ def start_task(self, task, rampart_group, dependent_tasks=None, instance=None):
         }
         dependencies = [{'type': get_type_for_model(type(t)), 'id': t.id} for t in dependent_tasks]
 
-        controller_node = None
-        if task.supports_isolation() and rampart_group.controller_id:
-            try:
-                controller_node = rampart_group.choose_online_controller_node()
-            except IndexError:
-                logger.debug("No controllers available in group {} to run {}".format(rampart_group.name, task.log_format))
-                return
+        try:
+            controller_node = Instance.choose_online_control_plane_node()
```

Comment:
All task execution now chooses *one* (random) online "control plane node" as the node through which work is submitted to the receptor mesh.  This model is similar to the isolated model, but in this new world, *all* task execution has a controller node (the node where we spoke to receptor via a local domain socket), and the execution node (the target receptor node where we routed work, based on available capacity).